modules/apps/segments/segments-web/test/js/components/criteria_builder/Conjunction.es.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {cleanup, render} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import Conjunction from '../../../../src/main/resources/META-INF/resources/js/components/criteria_builder/Conjunction.es';
+
+const conjunctions = [
+	{
+		label: 'AND',
+		name: 'and'
+	},
+	{
+		label: 'OR',
+		name: 'or'
+	}
+];
+
+describe('Conjunction', () => {
+	afterEach(cleanup);
+
+	it('renders with value in editable mode', () => {
+		const {
+			label: initialConjunctionLabel,
+			name: initialConjunctionName
+		} = conjunctions[0];
+
+		const {getAllByText} = render(
+			<Conjunction
+				conjunctionName={initialConjunctionName}
+				editing={true}
+				onSelect={() => {}}
+				supportedConjunctions={conjunctions}
+			/>,
+			{
+				baseElement: document.body
+			}
+		);
+
+		const [selectedConjunction] = getAllByText(initialConjunctionLabel);
+
+		expect(selectedConjunction.tagName).toBe('BUTTON');
```

Comment:
check label and name here?

Reply:
check if the button is not disabled?

Reply:
button is never disabled.
name is not present in the UI.